cdi-microservice-provider/src/main/java/io/silverware/microservices/providers/cdi/internal/DefaultMethodHandler.java

```diff
@@ -88,9 +87,7 @@ public Object invoke(final Method method, final Object... args) throws Exception
          }
       }
 
-      if (log.isDebugEnabled()) {
-         log.debug("Invocation of " + method);
-      }
+      log.debug("Invocation of " + method);
```

Comment:
This is the questionable skip of check - you construct a new string as a method parameter. This short-lived object creation should be omitted in case of a frequent code usage. I don't know whether this is the case, I am just saying...